alpine-nix-rails-nginx/etc/sv/rails

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -o errexit -o pipefail -o noglob -o noclobber -o nounset
+IFS=$'\n\t'
+
+cd /opt/rails
+exec sh -c "$(sed -n 's/#.*$//;/^$/d;s/^web:\s*//gp' Procfile)"
```

Comment:
Can we replace this in-place Procfile parser with a small ruby script like:

```
exec ruby --require yaml -e "exec YAML.load_file('Procfile').fetch('web')"
```

This way its easier to read/review. I realize that Procfiles are not meant to be YAML documents. But they seem to be a subset.